### ISSUE TITLE
Compare features btw ShEx and DCTAP?

### ISSUE THREAD

**Issue by tombaker** [rudof-project/rudof#152]
There are some out-of-date but nice tables for feature comparisons in https://github.com/weso/shaclex/issues/4#issue-177096065 - could be interesting to do something like that with regard to the subset of features in DCTAP supported by rudof?

**Comment by labra** [comment]
Indeed, we have a similar issue about ShEx features that are supported in rudof here: #8

About comparing DCTAP and ShEx, maybe I added another issue (#153) to collect what DCTAP features we support.

The comparison between DCTAP and ShEx features in general, should probably be done in either the DCTAP spec repo or the ShEx repo.

**Comment by tombaker** [comment]
@labra I have elaborated on this question in https://github.com/weso/NALShapes_Internal/issues/26#issuecomment-2338352460 .

Basically, I propose adapting DCTAP for our purposes. DCTAP was designed primarily for the use case of Application Profiles for metadata about textual data. We can select features that are more appropriate for tabular data.
